packages/extension-driver-snowflake/test/snowflakeDataSource.spec.ts

```diff
@@ -56,6 +59,75 @@ it('Data source should return correct rows and types', async () => {
   expect(columns[2]).toEqual({ name: 'C', type: 'boolean' });
 }, 10000);
 
+it('Data source should export parquet file correctly', async () => {
+  // Arrange
+  dataSource = new SnowflakeDataSource({}, '', [snow.getProfile('profile1')]);
+  await dataSource.activate();
+  const directory = `tmp`;
+  if (!fs.existsSync(directory)) {
+    fs.mkdirSync(directory);
+  }
+  // Act
+  const rawCount = 10 * 10000;
+  const exportOption: ExportOptions = {
+    sql: `SELECT seq4() as seq, uniform(1, 10, RANDOM(12)) as rand
+    FROM TABLE(GENERATOR(ROWCOUNT => ${rawCount})) v 
+    ORDER BY 1`,
+    profileName: 'profile1',
+    directory,
+  };
+  await dataSource.export(exportOption);
+  // Assert
+  const files = fs.readdirSync(directory);
+  expect(files.length).toBe(1);
+  expect(/parquet$/.exec(path.basename(files[0])));
```

Comment:
Well, the `expect(/parquet$/.exec(path.basename(files[0])))` actually may not really assert the result.

The value in the `expect(value)` is the actual value, we should call the method to do the assert.

Please use the [expect](https://jestjs.io/docs/expect) method to assert the result, e.g:

```ts
expect(files[0]).toMatch('/.parquet$/');
```